primitives/npos-elections/src/lib.rs

```diff
@@ -479,13 +352,40 @@ pub struct Support<AccountId> {
 	pub voters: Vec<(AccountId, ExtendedBalance)>,
 }
 
+#[cfg(feature = "mocks")]
```

Comment:
those feature are very easy to include by mistake, if eq_unordered is considered bad in production I would prefer to rename it `eq_unordered_expensive` or some more meaningful names

Reply:
This is unused code added in the past, I removed it.